api_types.ts

```diff
@@ -881,25 +881,25 @@ export enum PrecannedDateRange {
 /**
  * @hidden
```

Comment:
Should this be un-hidden now? I may have missed it in my PR that added documentation.

Reply:
I'll make a follow-up PR to unhide this and a few other things